doc/source/merging.rst

```diff
@@ -323,6 +323,11 @@ the name of the ``Series``.
           labels=['df1', 's1'], vertical=False);
    plt.close('all');
 
+.. note::
+
+   You can perform the same task using the DataFrame method
```

Comment:
this is not true. I would say if you want to add a single series to an existing dataframe, then you want to use assign, otherwise you want to use concat.

Reply:
It's true only in this exact case of concatenating a single series to a single dataframe. @obilodeau it'd be good to point out difference. `concat` is for concatenating an arbitrary number of pandas objects, while `.assign` is only defined on `DataFrame`, and only accepts 1-d argumnents. Something like

```
..  note::

   Since we're concatenating a Series to a DataFrame, we could have achieved the same
   output with :meth:`DataFrame.assign`. To concatenate an arbitrary number of pandas
   objects (DataFrame or Series), use ``concat``.
```